.Lib9c.Tests/Model/Skill/CombatTest.cs

```diff
@@ -136,5 +137,46 @@ public void Bleed()
             var skillInfo = Assert.Single(skill.SkillInfos);
             Assert.Equal(currentHP, skillInfo.Target!.CurrentHP);
         }
+
```

Comment:
What about putting a link to the issue?

```suggestion

        /// <summary>
        /// <see href="https://github.com/planetarium/lib9c/issues/2027">Issue #2027</see>.
        /// </summary>
```